src/RGLFields.hpp

```diff
@@ -318,20 +332,25 @@ inline std::vector<uint8_t> toRos2Fields(rgl_field_t type)
 
 inline std::vector<std::string> toRos2Names(rgl_field_t type)
 {
+	// clang-format off
 	switch (type) {
 		case XYZ_VEC3_F32: return {"x", "y", "z"};
 		case IS_HIT_I32: return {"is_hit"};
 		case IS_GROUND_I32: return {"is_ground"};
 		case ENTITY_ID_I32: return {"entity_id"};
 		case RAY_IDX_U32: return {"ray_idx"};
-		case INTENSITY_F32: return {"intensity"};
+		case INTENSITY_F32:
+		case INTENSITY_U8:
+			return {"intensity"};
```

Comment:
Because of these fields sharing a name, can there be a niche case somewhere (ROS 2 publisher?) that will cause problems in case of both intensities requested?

Reply:
This function is used by ROS2 publishers only and I see no blocker for having two fields with the same name.